pkg/kubectl/resource_printer.go

```diff
@@ -291,13 +293,25 @@ func (h *HumanReadablePrinter) unknown(data []byte, w io.Writer) error {
 	return err
 }
 
-func (h *HumanReadablePrinter) printHeader(columnNames []string, w io.Writer) error {
-	if _, err := fmt.Fprintf(w, "%s\n", strings.Join(columnNames, "\t")); err != nil {
+func (h *HumanReadablePrinter) printHeader(columnNames []string, withNamespace bool, w io.Writer) error {
+	columns := []string{}
+	if withNamespace {
```

Comment:
I agree, but it's a little comprecated to pass namespace name as value for each object.
Let me consider how to handle it for a while.